### PR TITLE
Meeting summaries for 2016-08-04

### PR DESCRIPTION
**Pull request by G1lius**

### CODE REVIEW

**Comment by kanzure** [comment]
ACK, with the exception of the mention of \#libconsensus -- it seems to be an empty channel, perhaps it was a typo? or maybe it moved?

**Comment by jonasschnelli** [comment]
ACK

**Comment by btcdrak** [comment]
@kanzure: I think it's a double has for the channel name `##libconsensus`

File: _posts/en/meetings/2016-08-04-meeting.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ 
+- [Link to this week logs](https://botbot.me/freenode/bitcoin-core-dev/2016-08-04/?msg=70789770&page=2)
+- [Meeting minutes by meetbot](http://www.erisian.com.au/meetbot/bitcoin-core-dev/2016/bitcoin-core-dev.2016-08-04-19.00.html)
+ 
+---
+ 
+## Notes / short topics
+
+- A few developers and miners met eachother in San Francisco where they went to Standford university to meet with Dan Boneh, a researcher in applied cryptography and computer security. Although not short term development related, [the transcript](http://diyhpl.us/wiki/transcripts/2016-july-bitcoin-developers-miners-meeting/dan-boneh/) made by kanzure has a very high signal-to-noise ratio. 
+- NicolasDorier made an IRC #libconsensus bikeshedding room about how to handle libconsensus. The design will be presented to the bigger group for feedback.
```
Comment (btcdrak) [inline _posts/en/meetings/2016-08-04-meeting.md]: s/#libconsensuss/##libconsensus/

**Pull request merged**